ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.j2

```diff
@@ -0,0 +1,112 @@
+# They indicate to the plugin to use their own builtin configuration to
+# connect to the local system.
+
+# NOTE: The configuration has a few required parameters. They are marked
+# with 'required'. Be sure to edit those to make this configuration work.
+
+# Tags can also be specified via a normal map, but only one form at a time:
+[tags]
+  # dc = "us-east-1"
+
+# Configuration for telegraf agent
+[agent]
+  # Default data collection interval for all plugins
+  interval = "10s"
+  # Rounds collection interval to 'interval'
+  # ie, if interval="10s" then always collect on :00, :10, :20, etc.
+  round_interval = true
+
+  # Default data flushing interval for all outputs. You should not set this below
+  # interval. Maximum flush_interval will be flush_interval + flush_jitter
+  flush_interval = "10s"
+  # Jitter the flush interval by a random amount. This is primarily to avoid
+  # large write spikes for users running a large number of telegraf instances.
+  # ie, a jitter of 5s and interval 10s means flushes will happen every 10-15s
+  flush_jitter = "0s"
+
+  # Run telegraf in debug mode
+  debug = {{ debug_mode }}
+  # Override default hostname, if empty use os.Hostname()
+  hostname = "{{ inventory_hostname }}"
+
+
+###############################################################################
+#                                  OUTPUTS                                    #
+###############################################################################
+
+[outputs]
+
+# Configuration for influxdb server to send metrics to
+[outputs.influxdb]
+  # The full HTTP endpoint URL for your InfluxDB instance
+  # Multiple urls can be specified for InfluxDB cluster support.
+  urls = [ "http://{{ inventory_hostname }}:8086" ] # required
+  # The target database for metrics (telegraf will create it if not exists)
+  database = "{{ target_db }}" # required
+  # Precision of writes, valid values are n, u, ms, s, m, and h
+  # note: using second precision greatly helps InfluxDB compression
+  precision = "s"
+
+  # Connection timeout (for the connection with InfluxDB), formatted as a string.
+  # If not provided, will default to 0 (no timeout)
+  timeout = "5s"
+  username = "{{ statsd_influxdb_user }}"
+  password = "{{ statsd_influxdb_passwd }}"
+  # Set the user agent for the POSTs (can be useful for log differentiation)
+  # user_agent = "telegraf"
+
+###############################################################################
+#                                  PLUGINS                                    #
+###############################################################################
+
+
+###############################################################################
+#                              SERVICE PLUGINS                                #
+###############################################################################
+
+# Statsd Server
+[statsd]
+  # Address and port to host UDP listener on
+  service_address = ":9125"
```

Comment:
Can we add a comment to explain this - "8125 has the actual stasd endpoint (repeaters). This is the port where the repeaters send the data to for each influxdb."